velox/functions/sparksql/Register.cpp

```diff
@@ -99,10 +99,6 @@ void registerFunctions(const std::string& prefix) {
       prefix + "rlike", re2SearchSignatures(), makeRLike);
   VELOX_REGISTER_VECTOR_FUNCTION(udf_regexp_split, prefix + "split");
 
-  // Subscript operators. See ExtractValue in complexTypeExtractors.scala.
-  VELOX_REGISTER_VECTOR_FUNCTION(udf_subscript, prefix + "getarrayitem");
```

Comment:
getarrayitem and getmapvalue are not Spark functions, but element_at is. Should we instead rename the function to element_at? 

https://spark.apache.org/docs/latest/api/sql/#element_at

Reply:
Just fixed it.

Reply:
Yes, I fix the UT.
1.  index start from 1 in spark.
2.  If index < 0, accesses elements from the last to the first.

Reply:
Hmm... this PR didn't change the function implementation, just the name. Why would this require a test change?

Reply:
Maybe I should remove SubscriptTest.cpp? Because element_at was already tested in prestosql\tests\ElementAtTest.cpp.